.tool-versions

```diff
@@ -1,4 +1,3 @@
-erlang 21.2
-elixir 1.8.1-otp-21
-nodejs 8.17.0
-python 3.8.5
```

Comment:
This was added in https://github.com/mbta/dotcom/pull/574 though admittedly I've never used the script and forgot about it until just now....

If we remove Python here (which I'm probably fine with?, as most devs won't use it anyway) we should probably add a note to the deploy script documentation about it.

Reply:
Nope, good catch. I'll put it back.